omniscidb/L0Mgr/L0Mgr.h

```diff
@@ -56,6 +58,35 @@ class L0Kernel;
 class L0CommandList;
 class L0CommandQueue;
 
+class L0DataFetcher {
+#ifdef HAVE_L0
+  static constexpr uint16_t GRAVEYARD_LIMIT{500};
```

Comment:
I'd avoid any potential problems by making it `size_t` instead of saving a couple bytes.